src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,14 +1,42 @@
 package core.basesyntax.service;
 
-import core.basesyntax.dao.StorageDao;
+import core.basesyntax.InvalidDataException;
 import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
-    private final StorageDao storageDao = new StorageDaoImpl();
+    private static final int VALID_AGE = 18;
+    private static final int VALID_LENGTH_PASS = 6;
 
     @Override
     public User register(User user) {
+        if (user == null) {
+            throw new InvalidDataException("User object can't be null");
```

Comment:
You should create your own exception 
![image](https://user-images.githubusercontent.com/32339836/223154328-02f75f44-941b-4124-af0d-7f9fc0012278.png)
Fix it in all places